src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,695 @@
+---
```

Comment:
This new doc needs to be added to the accounts.yml nav file for it appear in the nav.

Reply:
thank you!